metadata.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "beersy-snow_record",
-  "version": "0.1.5",
-  "author": "Abir Majumdar",
+  "name": "puppetlabs-servicenow_tasks",
```

Comment:
Huh, is this how the other `puppetlabs-<module_name>` modules do their metadata?

Reply:
I'll check again, but I think so.

Reply:
This looks consistent with the modules I looked at.

Reply:
Ah ok. How hard would it be to change the repo name to `puppetlabs-servicenow_tasks` ? Just feels weird having that naming inconsistency compared w/ the other repos.

Reply:
Sorry, I think I misled you. Other modules have the same setup as we do in the metadata where the name would be puppetlabs-servicenow_tasks, the repo name is servicenow_tasks, and the repo namespace is puppetlabs. So I think the way it is now is consistent with the rest of the modules up on the forge.

Reply:
Ah ok. Thanks for clarifying that.